lib/Controller/Display.php

```diff
@@ -178,14 +185,86 @@ function displayPage(Request $request, Response $response)
+     */
+    function displayMap(Request $request, Response $response)
+    {
+        $parsedQueryParams = $this->getSanitizer($request->getQueryParams());
+
+        $filter = [
+            'displayId' => $parsedQueryParams->getInt('displayId'),
+            'display' => $parsedQueryParams->getString('display'),
+            'useRegexForName' => $parsedQueryParams->getCheckbox('useRegexForName'),
+            'macAddress' => $parsedQueryParams->getString('macAddress'),
+            'license' => $parsedQueryParams->getString('hardwareKey'),
+            'displayGroupId' => $parsedQueryParams->getInt('displayGroupId'),
+            'clientVersion' => $parsedQueryParams->getString('clientVersion'),
+            'clientType' => $parsedQueryParams->getString('clientType'),
+            'clientCode' => $parsedQueryParams->getString('clientCode'),
+            'authorised' => $parsedQueryParams->getInt('authorised'),
+            'displayProfileId' => $parsedQueryParams->getInt('displayProfileId'),
+            'tags' => $parsedQueryParams->getString('tags'),
+            'exactTags' => $parsedQueryParams->getCheckbox('exactTags'),
+            'showTags' => true,
+            'clientAddress' => $parsedQueryParams->getString('clientAddress'),
+            'mediaInventoryStatus' => $parsedQueryParams->getInt('mediaInventoryStatus'),
+            'loggedIn' => $parsedQueryParams->getInt('loggedIn'),
+            'lastAccessed' => ($parsedQueryParams->getDate('lastAccessed') != null) ? $parsedQueryParams->getDate('lastAccessed')->format('U') : null,
+            'displayGroupIdMembers' => $parsedQueryParams->getInt('displayGroupIdMembers'),
+            'orientation' => $parsedQueryParams->getString('orientation'),
+            'commercialLicence' => $parsedQueryParams->getInt('commercialLicence'),
+            'folderId' => $parsedQueryParams->getInt('folderId'),
+            'logicalOperator' => $parsedQueryParams->getString('logicalOperator'),
+        ];
+
+        // Get a list of displays
+        $displays = $this->displayFactory->query($this->gridRenderSort($parsedQueryParams), $this->gridRenderFilter($filter, $parsedQueryParams));
+        $results = [];
+        $status = [
+            '1' => 'Up to date',
+            '2' => 'Downloading',
+            '3' => 'Out of date'
+        ];
+        foreach ($displays as $display) {
+         $geo = new Point([(double)$display->longitude, (double)$display->latitude]);
+
+            $results[] = new Feature($geo, [
+                'display' => $display->display,
+                'status' => $display->mediaInventoryStatus ? $status[$display->mediaInventoryStatus] : 'Unknown',
+                'orientation' => ucwords($display->orientation),
+                'displayId' => $display->getId(),
+                'licensed' => $display->licensed,
+            ]);
+        }
+
+        return $response->withJson(new FeatureCollection($results));
```

Comment:
Ah, this explains why you have `jmikola/geojson` included, but not the other one.